code/tables/output/robustness_city.tex

```diff
@@ -316,7 +316,7 @@
 Koreatown           &      -71.37\sym{***}&                     &                     &                     &                     &                     &                     \\
                     &     (4.758)         &                     &                     &                     &                     &                     &                     \\
 [1em]
-La Ca�ada Flintridge&      -79.27\sym{***}&                     &                     &                     &                     &                     &                     \\
+La Cañada Flintridge&      -79.27\sym{***}&                     &                     &                     &                     &                     &                     \\
```

Comment:
we shouldn't even have all these FEs in here, it should be keep(city coefficients) in the esttab command